scripts/monitoring/auto_scale_workers.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```

Comment:
Can we have all the imports starting with `from` after the `import <package>` statements?